### DISCUSSION TITLE
Switching from Bitwarden_RS to VaultWarden, potential issues?

### DISCUSSION THREAD

**Discussion by MostHated** [dani-garcia/vaultwarden#1660] [General]
Hey there,
    I am still running Bitwarden_RS via docker and was figuring it is probably a good idea to swap over to the new setup. I can't say I have ever moved from one image to another like this, so I was wondering, is it safe to simply swap out the docker image name to the new one or are there major changes that need to be accounted for in the upgrade process?

Thanks,
-MH

**Comment by iceman73** [comment]
Just swap the Image you are using and it will be fine. The author Outlined the steps here:

https://github.com/dani-garcia/vaultwarden/discussions/1642

**Comment by dhruv130797** [comment]
I'm using Bitwarden_RS for my personal use and wanted to be assured that I won't loose any data if updating just the image name in portainer before deploying this new image.

**Reply by iceman73** [reply]
Did you read the Link i provided. As Long as you have your Data on a Volume you will be fine to follow those instructions

**Reply by dhruv130797** [reply]
Okay Thanks!
